pipeline.py

```diff
@@ -26,10 +31,36 @@ def predict(self):
         #TODO @ZhouyuanYuan
         pass
 
-    def process(self):
-        """Process binary files (not to be confused with model preprocessing"""
-        #TODO @TarujGoyal
-        pass
+    def process(self, date, root="/data6/phytoplankton-db/hab_in_vitro"):
```

Comment:
set the default value for `root` by importing a set `data_dir`
Also root_dir should be `/data6/phytoplankton-db/hab_in_vitro/images`
i.e.

```
pipeline.py
# Project level imports
from config.config import Config

# Module level constants
DEFAULT_ENV = CONST.PROD_ENV
opt = Config(DEFAULT_ENV)
...
..
.
def process(self, date, root=opt.data_dir)
```

Reply:
also can you set up the main() and set the pipeline.py to accept a date argument and feed that into pipeline.process()